src/pages/[platform]/build-a-backend/storage/storage-v5-to-v6-migration-guide/index.mdx

```diff
@@ -55,56 +55,55 @@ The `Storage.put` API has been renamed to `uploadData` in v6. There are a few ch
       // AWSS3 (Default) Provider
       key: string
       object: {
-        ACL?: string; // Not supported in v6
-        CacheControl?: string;
+        ACL?: string; // Deprecated in V6
```

Comment:
Nit: Kind of feel we should err towards "removed" instead of deprecated for the same reason?